Puppetfile.tracking

```diff
@@ -26,7 +26,7 @@ mod 'rubygem-simp_cli',
 moduledir 'src/puppet/modules'
 
 mod 'bfraser-grafana',
-  :git => 'https://github.com/simp/puppet-grafana',
```

Comment:
This should still be our fork. If master needs to be updated that's another ticket.